### PR TITLE
Add camera position in facing property

### PR DESCRIPTION
**Pull request by KarlosQ**
- Add camera position in device list. Now we can get it in facing property. 
- Fix readme
- Add RTCView explication with props

### CODE REVIEW

File: ios/RCTWebRTC/WebRTCModule+RTCMediaStream.m [hunk 1]
```diff
@@ -121,6 +121,7 @@ - (RTCVideoTrack *)createVideoTrack:(NSDictionary *)constraints {
     NSArray *videoDevices = [AVCaptureDevice devicesWithMediaType:AVMediaTypeVideo];
     for (AVCaptureDevice *device in videoDevices) {
         [devices addObject:@{
+                             @"facing": device.position,
```
Comment (saghul) [inline ios/RCTWebRTC/WebRTCModule+RTCMediaStream.m]: AFAIS Position is an NSInteger, we need a "front" or "back" string instead: https://developer.apple.com/documentation/avfoundation/avcapturedeviceposition?language=objc

Comment (KarlosQ) [reply ios/RCTWebRTC/WebRTCModule+RTCMediaStream.m]: Thanks you.
It's correct now? I can't try in a real phone actually, so please try it if you can.

File: ios/RCTWebRTC/WebRTCModule+RTCMediaStream.m [hunk 2]
```diff
@@ -120,8 +120,14 @@ - (RTCVideoTrack *)createVideoTrack:(NSDictionary *)constraints {
     NSMutableArray *devices = [NSMutableArray array];
     NSArray *videoDevices = [AVCaptureDevice devicesWithMediaType:AVMediaTypeVideo];
     for (AVCaptureDevice *device in videoDevices) {
+        NSString *position = @"";
+        if (device.position == 1) {
```
Comment (saghul) [inline ios/RCTWebRTC/WebRTCModule+RTCMediaStream.m]: Please don't use numbers, there are defined constants.

Comment (KarlosQ) [reply ios/RCTWebRTC/WebRTCModule+RTCMediaStream.m]: Ok, sorry. Can you check it now?

**Comment by saghul** [comment]
Nice work, thanks!

**Pull request merged**